src/background/controllers/accountController.ts

```diff
@@ -351,9 +361,16 @@ export default class AccountController extends IController {
     await this.loggedInAccount.wallet.getInfo();
     const newBalance = this.loggedInAccount.wallet.info!.balance;
 
-    // if the balance has changed, update the inpageAcct for all windows
-    if (existingBalance !== newBalance) {
-      this.main.inpageAccount.sendInpageAccountAllPorts();
+    /**
+     * If this.justLoggedIn === true, then we only just logged in, and since we are
+     * already calling inpageAccount.sendInpageAccountAllPorts() on login, we don't want
+     * to call it a second time here.
+     * Otherwise, if the balance has changed, update the inpageAcct for all windows
+     */
+    if (this.justLoggedIn) {
+      this.justLoggedIn = false;
```

Comment:
adding this flag to prevent the double update seems like a bandaid fix. the double update happens because of:
1. getInfo call when we start polling
2. getInfo call when logging in
right?

why dont we just remove one of the `getInfo` calls. remove the login getInfo call? since we know that the polling should happen so we can get the balance. and polling is continuously happening in the bg now so we just need to make sure that the polling gets started again when they re-login in.